diff --git c/gtkgreet/window.c i/gtkgreet/window.c
index dcde118..970925d 100644
--- c/gtkgreet/window.c
+++ i/gtkgreet/window.c
@@ -94,6 +94,8 @@ void window_setup_question(struct Window *ctx, enum QuestionType type, char* que
             if (type == QuestionTypeSecret) {
                 gtk_entry_set_input_purpose((GtkEntry*)ctx->input_field, GTK_INPUT_PURPOSE_PASSWORD);
                 gtk_entry_set_visibility((GtkEntry*)ctx->input_field, FALSE);
+            } else if ( type == QuestionTypeInitial ) {
+                gtk_entry_set_text( (GtkEntry*)ctx->input_field, "teapot" );
             }
             g_signal_connect(ctx->input_field, "activate", G_CALLBACK(action_answer_question), ctx);
             gtk_widget_set_size_request(ctx->input_field, 384, -1);
