diff --git i/gtkgreet/window.c w/gtkgreet/window.c
index dcde118..a5d96ba 100644
--- i/gtkgreet/window.c
+++ w/gtkgreet/window.c
@@ -85,9 +85,9 @@ void window_setup_question(struct Window *ctx, enum QuestionType type, char* que
         case QuestionTypeInitial:
         case QuestionTypeVisible:
         case QuestionTypeSecret: {
-            GtkWidget *label = gtk_label_new(question);
-            gtk_widget_set_halign(label, GTK_ALIGN_END);
-            gtk_container_add(GTK_CONTAINER(question_box), label);
+            // GtkWidget *label = gtk_label_new(question);
+            // gtk_widget_set_halign(label, GTK_ALIGN_END);
+            // gtk_container_add(GTK_CONTAINER(question_box), label);

             ctx->input_field = gtk_entry_new();
             gtk_widget_set_name(ctx->input_field, "input-field");
@@ -103,9 +103,9 @@ void window_setup_question(struct Window *ctx, enum QuestionType type, char* que
         }
         case QuestionTypeInfo:
         case QuestionTypeError: {
-            GtkWidget *label = gtk_label_new(question);
-            gtk_widget_set_halign(label, GTK_ALIGN_END);
-            gtk_container_add(GTK_CONTAINER(question_box), label);
+            // GtkWidget *label = gtk_label_new(question);
+            // gtk_widget_set_halign(label, GTK_ALIGN_END);
+            // gtk_container_add(GTK_CONTAINER(question_box), label);
             break;
         }
     }
