diff --git i/pam_kwallet.c w/pam_kwallet.c
index 841e766..843a93f 100644
--- i/pam_kwallet.c
+++ w/pam_kwallet.c
@@ -62,7 +62,7 @@
 const static char *kdehome = NULL;
 const static char *kwalletd = NULL;
 const static char *socketPath = NULL;
-static int force_run = 0;
+static int force_run = 1;
 
 const static char * const kwalletPamDataKey = "kwallet5_key";
 const static char * const logPrefix = "pam_kwallet5";
